content/reference-rippled.md

```diff
@@ -8345,6 +8368,11 @@ The `info` object may have some arrangement of the following fields:
 | load.job\_types | Array | *Admin only* Information about the rate of different types of jobs the server is doing and how much time it spends on each. |
 | load.threads | Number | *Admin only* The number of threads in the server's main job pool. |
 | load\_factor | Number | The load factor the server is currently enforcing, as a multiplier on the base transaction cost. The load factor is determined by the highest of the individual server's load factor, cluster's load factor, and the overall network's load factor. **Note:** This `load_factor` is calculated as the ratio of the `load_factor` and the `load_base` that are reported by the [`server_state` command](#server-state) |
+| load\_factor\_local | Number | (May be omitted) Current multiplier to the [transaction cost][] based on load to this server. |
```

Comment:
I'm not 100% sure on the logic of when it can be omitted, but I don't think it's as simple as "when there's no multiplier" so I'd rather just leave it vague.